src/sage/combinat/skew_tableau.py

```diff
@@ -1139,6 +1140,81 @@ def to_list(self):
         """
         return [list(row) for row in self]
 
+    def row_stabilizer(self):
+        """
+        Return the PermutationGroup corresponding to the row stabilizer of
+        ``self``.
+
+        This assumes that every integer from `1` to the size of ``self``
+        appears exactly once in ``self``.
+
+        EXAMPLES::
+
+            sage: # needs sage.groups
+            sage: rs = SkewTableau([[None,1,2,3],[4,5]]).row_stabilizer()
+            sage: rs.order() == factorial(3)*factorial(2)
```

Comment:
```suggestion
            sage: rs = SkewTableau([[None,1,2,3], [4,5]]).row_stabilizer()
            sage: rs.order() == factorial(3) * factorial(2)
```
And perhaps similar spacing between the lists. At least, it should be consistent (see the next `rs = ...` below).